### PR TITLE
Uplifting label file for clang 13

### PR DESCRIPTION
**Pull request by dkrupp**
Added labels for new checkers in clang13
for clang static analyzer and clang tidy.

### CODE REVIEW

File: config/labels/analyzers/clang-tidy.json [hunk 1]
```diff
@@ -522,6 +522,15 @@
       "sei-cert:oop54-cpp",
       "severity:MEDIUM"
     ],
+    "bugprone-unhandled-exception-at-new": [
+      "doc_url:https://clang.llvm.org/extra/clang-tidy/checks/bugprone-unhandled-exception-at-new.html",
+      "guideline:sei-cert",
+      "profile:extreme",
+      "profile:security",
+      "profile:sensitive",
```
Comment (dkrupp) [inline config/labels/analyzers/clang-tidy.json:530]: take it out as there is no projects with results to this checker

File: config/labels/analyzers/clangsa.json [hunk 2]
```diff
@@ -306,9 +317,6 @@
       "doc_url:https://clang.llvm.org/docs/analyzer/checkers.html#alpha-webkit-uncountedlocalvarschecker",
       "profile:extreme"
     ],
-    "altera-struct-pack-align": [
```
Comment (csordasmarton) [inline config/labels/analyzers/clangsa.json:309]: Why did you remove this? Users who are using old clang versions will still have this checker.

File: config/labels/analyzers/clang-tidy.json [hunk 3]
```diff
@@ -5645,6 +5688,10 @@
       "doc_url:https://clang.llvm.org/extra/clang-tidy/checks/readability-container-size-empty.html",
       "severity:STYLE"
     ],
+    "readability-container-data-pointer": [
+      "doc_url:https://clang.llvm.org/extra/clang-tidy/checks/readability-data-pointer.html",
```
Comment (csordasmarton) [inline config/labels/analyzers/clang-tidy.json:5689]: I get 404 error when I visit this url. In this case it should be removed or please fix the url.

**Review by csordasmarton — CHANGES_REQUESTED** [review]

File: config/labels/analyzers/clangsa.json [hunk 4]
```diff
@@ -233,8 +233,23 @@
     ],
     "alpha.security.cert.pos.34c": [
       "doc_url:https://clang.llvm.org/docs/analyzer/checkers.html#alpha-security-cert-pos-34c",
+      "profile:default",
+      "profile:sensitive",
+      "profile:security",
+      "profile:extreme",
+      "severity:high",
+      "sei-cert:pos34-c"
+
```
Comment (csordasmarton) [inline config/labels/analyzers/clangsa.json:242]:
```suggestion
```

File: config/labels/analyzers/clangsa.json [hunk 5]
```diff
@@ -233,8 +233,23 @@
     ],
     "alpha.security.cert.pos.34c": [
       "doc_url:https://clang.llvm.org/docs/analyzer/checkers.html#alpha-security-cert-pos-34c",
+      "profile:default",
+      "profile:sensitive",
+      "profile:security",
+      "profile:extreme",
+      "severity:high",
```
Comment (csordasmarton) [inline config/labels/analyzers/clangsa.json:240]:
```suggestion
      "severity:HIGH",
```

**Review by csordasmarton — CHANGES_REQUESTED** [review]

**Review by csordasmarton — APPROVED** [review]
LGTM!

File: config/labels/analyzers/clangsa.json (hunk 2 above)
Comment (whisperity) [reply config/labels/analyzers/clangsa.json:309]: Not just old versions, I think this check is still part of the upstream distribution: https://github.com/llvm/llvm-project/blob/91f4650ebb057de7f6ed6d038985b2d53d144e15/clang-tools-extra/clang-tidy/altera/StructPackAlignCheck.cpp

(Then again, it is a very guidelines or platform specific check.)
